desktop.bundles/index/index.bemjson.js

```diff
@@ -4,8 +4,9 @@ module.exports = {
     favicon: '/favicon.ico',
     head: [
         { elem: 'meta', attrs: { name: 'description', content: '' } },
-        { elem: 'meta', attrs: { name: 'viewport', content: 'width=device-width, initial-scale=1' } },
-        { elem: 'css', url: 'index.min.css' }
+        { elem: 'meta', attrs: { name: 'viewport', content: 'width=device-width, initial-scale=1, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no' } },
+        { elem: 'css', url: 'index.min.css' },
+        '<script src="http://ajax.googleapis.com/ajax/libs/jquery/1/jquery.min.js"></script>'
```

Comment:
move it to `scripts` array as
```js
scripts: [
    { elem: 'js', url: 'http://ajax.googleapis.com/ajax/libs/jquery/2.2.2/jquery.min.js' },
    { elem: 'js', url: 'index.min.js' }
]
```

I think version `2` is better. And I'd also use specific `2.2.2` instead of just major.